src/pages/index.page.tsx

```diff
@@ -28,33 +33,21 @@ const Home: NextPage = () => {
         </p>
 
         <div className={styles.grid}>
-          <a href="https://nextjs.org/docs" className={styles.card}>
-            <h2>Documentation &rarr;</h2>
-            <p>Find in-depth information about Next.js features and API.</p>
-          </a>
-
-          <a href="https://nextjs.org/learn" className={styles.card}>
-            <h2>Learn &rarr;</h2>
-            <p>Learn about Next.js in an interactive course with quizzes!</p>
-          </a>
-
-          <a
-            href="https://github.com/vercel/next.js/tree/canary/examples"
-            className={styles.card}
-          >
-            <h2>Examples &rarr;</h2>
-            <p>Discover and deploy boilerplate example Next.js projects.</p>
-          </a>
-
-          <a
-            href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-            className={styles.card}
-          >
-            <h2>Deploy &rarr;</h2>
-            <p>
-              Instantly deploy your Next.js site to a public URL with Vercel.
-            </p>
-          </a>
+          {posts.map((post) => (
+            <Link href="/posts/[id]" as={`/posts/${post.slug}`} key={post.slug}>
+              <a href={`/posts/${post.slug}`} className={styles.card}>
+                <h2>{post.title}</h2>
+                <Image
+                  src={post.coverImage}
+                  alt={post.title}
+                  width={480}
+                  height={320}
+                  layout="responsive"
```

Comment:
```suggestion
                  layout="responsive"
                  objectFit="contain"
```
`layout="responsive"` affects the aspect ratio, in this case, should we consider passing the `objectFit="contain"` props too to `Image`, that'll result as:

Currently | With `objectFit="cover"`
--- | ---
<img width="649" alt="image" src="https://user-images.githubusercontent.com/39977805/170035841-c64ae91c-ef50-44df-a765-0847a31313f9.png"> | <img width="677" alt="image" src="https://user-images.githubusercontent.com/39977805/170035708-d70983e1-1fd7-43ad-9270-9bb9360929d7.png">

Reply:
Fixed in https://github.com/nimblehq/til-web/pull/8/commits/b27fcb650a803e5a0588e94bb9b0f3e5267786a1